apps/src/JsInterpreterLogger.js

```diff
@@ -0,0 +1,59 @@
+/** @file Observes a JSInterpreter and logs to the designated browser console. */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+ eqeqeq: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+'use strict';
+
+var Observer = require('./Observer');
+
+/**
+ * Observer responsible for logging to the provided browser console when
+ * the interpreter it is observing raises log-worthy events.
+ * @constructor
+ * @param {Console} window console API
+ */
+var JsInterpreterLogger = module.exports = function (outputConsole) {
```

Comment:
This name is pretty terrible.  Suggestions?  I wanted a less entangled and more reusable way to echo logging to the browser console.